### DISCUSSION TITLE
Should `modifySchemaContent` update the linter?

### DISCUSSION THREAD

**Discussion by microhobby** [redhat-developer/vscode-yaml#899] [Q&A]
I'm writing an extension to update some `enum` in the schema by files created and deleted from workspace. But when calling the `modifySchemaContent` the linter squiggles does not update. This should be updated?

**Comment by gorkem** [comment]
Hmm. It looks like `modifySchemaContent` only changes the schemas. We probably should have the validation run again. Sounds like this should be converted to an issue

**Reply by microhobby** [reply]
@gorkem done https://github.com/redhat-developer/vscode-yaml/issues/1062
